contrib/podmanimage/machine/Containerfile

```diff
@@ -0,0 +1,15 @@
+FROM quay.io/centos-bootc/fedora-bootc:eln
+
+RUN dnf -y update; \
+rpm --setcaps shadow-utils 2>/dev/null; \
+dnf -y install podman subscription-manager crun crun-wasm crun-krun chrony \
+skopeo buildah; dnf -y clean all; \
+systemctl enable podman.service; \
+printf "\nconfdir /etc/chrony.d\n" >> /etc/chrony.conf
+
+ADD usr usr
+ADD etc etc
+
+RUN groupadd -g 501 core; useradd -u 501 -g 501 core
```

Comment:
the user cannot be made in containerfile due to `podman machine init --username`

Reply:
also, if you want to push forward on this, should be user 1000

Reply:
I would be fine with removing this, but do we want to set the default user for the case where these is no specified --username and then allow the user to modify it. Might be a case where want to use sysusers.